ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -201,6 +201,22 @@ RUN git clone https://github.com/kanaka/noVNC /home/user/.novnc/ \
  && cd /home/user/.novnc \
  && npm install \
  && node ./utils/use_require.js --as commonjs --with-app
+ 
+# Install dependencies to enable codeintel (https://github.com/c9/c9.ide.language.codeintel)
+RUN easy_install pip
+ && pip install -U pip
+ && pip install -U virtualenv
+ && virtualenv --python=python2 $HOME/.c9/python2
+ && source $HOME/.c9/python2/bin/activate
+ && apt-get update
+ && apt-get install python-dev
+ && mkdir /tmp/codeintel
+ && pip download -d /tmp/codeintel codeintel==0.9.3
+ && cd /tmp/codeintel
+ && tar xf CodeIntel-0.9.3.tar.gz
+ && mv CodeIntel-0.9.3/SilverCity CodeIntel-0.9.3/silvercity
```

Comment:
Interesting, why are you renaming `SilverCity` to `silvercity`?

Reply:
I don't know much, I have to check https://github.com/c9/c9.ide.language.codeintel, that's what they advocate to do